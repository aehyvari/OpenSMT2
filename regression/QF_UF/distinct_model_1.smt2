(set-logic QF_UF)
(declare-sort U 0)
(declare-fun c1 () U)
(declare-fun c2 () U)
(declare-fun c3 () U)
(assert (not (= c1 c2)))
(assert (not (= c1 c3)))
(assert (not (= c2 c3)))
(check-sat)
(get-value ((distinct c1 c2 c3)))
