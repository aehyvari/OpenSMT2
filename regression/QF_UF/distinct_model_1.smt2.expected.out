sat
(((distinct c1 c2 c3) true))
